src/ios/CDVWKInAppBrowser.m

```diff
@@ -1067,6 +1067,12 @@ - (UIStatusBarStyle)preferredStatusBarStyle
     NSString* statusBarStylePreference = [self settingForKey:@"InAppBrowserStatusBarStyle"];
     if (statusBarStylePreference && [statusBarStylePreference isEqualToString:@"lightcontent"]) {
         return UIStatusBarStyleLightContent;
+    } else if (statusBarStylePreference && [statusBarStylePreference isEqualToString:@"darkcontent"]) {
+        if (@available(iOS 13.0, *)) {
+            return  UIStatusBarStyleDarkContent;
```

Comment:
```suggestion
            return UIStatusBarStyleDarkContent;
```

a bit nitpicky, but there is a space too much.

Reply:
Good catch